tmt/steps/provision/bootc.py

```diff
@@ -410,6 +429,9 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
         built_image.rename(renamed_image)
         data.image = f"file://{renamed_image}"
 
+        if data.build_disk_image_only:
```

Comment:
Does it makes sense to print path to the generated image?  
If one runs just `tmt run provision -h bootc ... --build-disk-image-only` path is not printed.

Reply:
Thank you for the review! Yes, make sense. Updated.